sdc/rabbit/async_consumer.py

```diff
@@ -0,0 +1,387 @@
+import logging
```

Comment:
Done - sdx-common was only getting pulled in to format the pika logging in the unit tests. It is no longer required.